src/main/java/jalau/at18/azul/view/TilesSend.java

```diff
@@ -0,0 +1,21 @@
+package jalau.at18.azul.view;
+
+import jalau.at18.azul.BagSender;
+import javax.swing.DefaultListModel;
+import javax.swing.JList;
+import javax.swing.JPanel;
+
+public class TilesSend extends JPanel {
+    private JList<String> countryList;
+    public TilesSend(int posx, int posy, int width, int height) {
+        //setBounds(posx, posy, width, height);
```

Comment:
Remove commented line if it's not needed

Reply:
already remove the comment